src/Symfony/Component/Mailer/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^7.2.5",
         "egulias/email-validator": "^2.1.10",
         "psr/log": "~1.0",
-        "symfony/event-dispatcher": "^4.4|^5.0",
+        "symfony/event-dispatcher": "^5.0",
```

Comment:
is this really required? can you please remind us why if yes?

Reply:
If we allowed EventDispatcher 4.4, we could receive a dispatcher object that expects the 4.2-style parameter order. We would still need the now-deprecated proxy in that case.

If we want to maintain 4.4 compatibility here, I could apply the check I've used in the notifier component.

Reply:
What about using the class_exists check everywhere and keep compat with v4.4?
I don't like narrowing version constraints :)

Reply:
Fair enough.